lighthouse-core/test/gather/computed/trace-of-tab-test.js

```diff
@@ -54,7 +54,7 @@ describe('Trace of Tab computed artifact:', () => {
   describe('finds correct FMP', () => {
     it('if there was a tracingStartedInPage after the frame\'s navStart', async () => {
       const trace = await TraceOfTab.compute_(lateTracingStartedTrace);
-      assert.equal(trace.startedInPageEvt.ts, 29343544280);
```

Comment:
building on @brendankenny's earlier point do we ever really use the `ts` anywhere other than the tests here? 

feels weird to have `ts` on a property called `*Ids` (my fault I know 😄) and as he pointed out we really just want to make sure that we're identifying the mainFrame correctly